src/components/post/_post.scss

```diff
@@ -5,7 +5,7 @@
     text-decoration: none;
 
     .post__image {
-      opacity: 0.75;
+      opacity: .75;
```

Comment:
Noticing a lot of `opacity: .75;`s. Is it worth considering a global `$opaque` variable(s)?